src/app/zap-templates/templates/chip/python-ChipDeviceController-ClusterCommands.zapt

```diff
@@ -0,0 +1,52 @@
+{{> header}}
+
+#include <core/CHIPCore.h>
+
+#include <app/CommandSender.h>
+#include <app/InteractionModelEngine.h>
+#include <app/im-encoder.h>
+#include <support/ReturnMacros.h>
+
+#include <controller/CHIPClusters.h>
+#include <controller/CHIPDevice.h>
+
+using namespace chip;
+using namespace chip::app;
+
+namespace chip {
+namespace app {
+CommandSender * GetCommandSender();
+namespace cluster {
+
+// Empty command handlers
+{{#chip_server_clusters}}
+namespace {{asCamelCased name false}} {
+{{#chip_server_cluster_commands}}
+void Handle{{asType name}}CommandReceived(chip::TLV::TLVReader & aReader, chip::app::Command * apCommandObj) {}
+{{/chip_server_cluster_commands}}
+} // namespace {{asCamelCased name false}}
+
+{{/chip_server_clusters}}
```

Comment:
Similarly to `im-encoder-src.zapt` you want have something that iterates over client clusters, not servers.

Reply:
We simplily dont have such thing like "chip_client_clusters", also they should be server clusters since they are intend to be decoders, although they will be refactored by follow up prs.

Reply:
I guess I was wrong for `im-encoder-src.zapt`. 

But on the python side, do you expect to receive on/off commands or do you expect to receive the response from the on/off command you have sent ?
If you expect responses, you should iterate over the client clusters.

I guess we can discuss over that on slack, but for the sake of this PR, I believe this code is unused ? If so, can we just remove it ?